modules/ROOT/pages/until-successful-scope.adoc

```diff
@@ -23,8 +23,8 @@ The Until Successful scope provides fields described in this table:
 [%header,cols="1,3"]
 |===
 |Attribute |Description
-|Max Retries (`maxRetries`) |Specifies the maximum number of retries that are attempted. An error message looks like this:  `Message: 'until-successful' retries exhausted.` The Mule error type is `MULE:RETRY_EXHAUSTED`.
-|Milliseconds Between Retries (`millisBetweenRetries`) |Specifies the minimum interval between two attempts to process, in milliseconds. The actual interval depends on the previous execution, but should not exceed twice this number. The default value is 60000 milliseconds (one minute).
+|Max Retries (`maxRetries`) |Specifies the maximum number of retries that are attempted. This attribute can be either a number or an expression that resolves to a number. An error message looks like this:  `Message: 'until-successful' retries exhausted.` The Mule error type is `MULE:RETRY_EXHAUSTED`.
+|Milliseconds Between Retries (`millisBetweenRetries`) |Specifies the minimum interval between two attempts to process, in milliseconds. The actual interval depends on the previous execution, but should not exceed twice this number. The default value is 60000 milliseconds (one minute). This attribute can be either a number or an expression that resolves to a number. 
```

Comment:
Although this is more complex in wording, I think it is clearer:
```suggestion
|Milliseconds Between Retries (`millisBetweenRetries`) |Specifies, in milliseconds, the minimum interval between two retries. The actual interval depends on the previous execution, but it should not exceed twice this number. The default value is 60000 milliseconds (one minute). This attribute can be either a number or an expression that resolves to a number. 
```

Reply:
Or have I changed meaning? Is an original attempt and one retry also considered "between retries"?